plugins/modules/nd_federation_member.py

```diff
@@ -0,0 +1,271 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Anvitha Jain (@anvjain) <anvjain@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: nd_federation_member
+version_added: "0.3.0"
+short_description: Setup multi-cluster configuration on Cisco Nexus Dashboard (ND).
+description:
+  - Connects multiple clusters together for a single pane of glass view and administration of the clusters and their sites, services, and configurations.
+  - M(cisco.nd.nd_federation_member) can only be used with python 3.7 and higher.
+  - Can only be used with Nexus Dashboard versions >= 2.3(2d).
+  - The ND version of the local cluster must be greater than or equal to the ND versions of the clusters being added.
+author:
+  - Anvitha Jain (@anvjain)
+options:
+  clusters:
+    description:
+      - The IP address of the cluster.
+    type: list
+    elements: dict
+    suboptions:
+      hostname:
+        description:
+          - The IP address of the federation member/cluster.
+        type: str
+        required: true
+        aliases: [ cluster_ip, hostname, ip_address, federation_member ]
```

Comment:
Just got to remove the duplicated alias to fix sanity. :)
```suggestion
        aliases: [ cluster_ip, ip_address, federation_member ]
```

Reply:
changed

Reply:
fixed